modules/contact/test/tests/fieldsplit/tests

```diff
@@ -1,11 +1,18 @@
 [Tests]
-  issues = '#7485'
-  design = 'FieldSplitPreconditioner.md'
+  issues = '#7485 #14791'
+  design = 'FieldSplitPreconditioner.md NormalMortarLMMechanicalContact.md TangentialMortarLMMechanicalContact.md'
   [./2block-3D]
     type = 'Exodiff'
     input = '2blocks3d.i'
     exodiff = '2blocks3d_out.e'
     max_parallel = 4
     requirement = 'The system shall allow for split preconditioning based on contact surfaces.'
   [../]
+  [./mortar-FS]
+    type = 'Exodiff'
+    input = 'frictional_mortar_FS.i'
+    exodiff = 'frictional_mortar_FS_out.e'
+    abs_zero = 1e-8
+    requirement = 'The mortar contact problem shall converge reasonably fast with split preconditioning.'
```

Comment:
“The system shall allow split preconditioning of frictional mortar contact.”

I don’t think we’re prepared to comment on its efficiency yet.

Reply:
> “The system shall allow split preconditioning of frictional mortar contact.”
> 
> I don’t think we’re prepared to comment on its efficiency yet.

Yea I agree. This example shows FS works with mortar contact. The efficiency is yet to be investigated and improved.